tsrc/test/cli/test_init.py

```diff
@@ -265,3 +265,26 @@ def test_several_remotes(
     )
     assert rc == 0, output
     assert output == "git@upstream.com"
+
+
+def test_singular_remote(
+    tsrc_cli: CLI, git_server: GitServer, workspace_path: Path
+) -> None:
+    foo_url = git_server.add_repo("foo")
```

Comment:
I have a better idea for those two tests: use an url that _cannot_ work.

You are writing a feature with a clear scenario in mind, use it in the tests!

```python
   git_server.manifest.set_repo_remotes(
        "foo",
        [("origin", foo_url),
         ("vpn", "foocorp.vpn/foo")]
)
```

This way, you don't have to assert as much because any bug in the production code will make the test fail if 
the vpn url is used.

Reply:
Great point!